GeneratedLibraries.proj

```diff
@@ -0,0 +1,96 @@
+<Project ToolsVersion="12.0" DefaultTargets="Clean;UpdateDiscoveryDocuments;Generate;Build;Package" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <!--
+    This project file requires that 'nuget' and 'python' (version 2)
+    be in the user's %PATH%.
+  -->
+
+  <Target Name="Clean">
+    <!-- Remove downloaded or generated artifacts. -->
+    <RemoveDir Directories="DiscoveryJson;Src/Generated;NuPkgs/Generated" />
+  </Target>
+
+  <Target Name="UpdateDiscoveryDocuments">
+    <MakeDir Directories="DiscoveryJson" />
+    <Exec Command="python get_discovery_documents.py --destination_dir .\DiscoveryJson" />
+  </Target>
+
+  <Target Name="Generate">
+    <PropertyGroup>
+      <_GenerateLibraryTool>ClientGenerator\generate_library.cmd</_GenerateLibraryTool>
+    </PropertyGroup>
+
+    <ItemGroup>
+      <_DiscoveryDocument Include="DiscoveryJson\*.json" />
+    </ItemGroup>
+
+    <MakeDir Directories="Src\Generated" />
```

Comment:
Are we going to publish each revision separately? (IMO it's recommended).
Then we won't just have the latest version we will all revision checked in...